.gitignore

```diff
@@ -14,3 +14,4 @@ DVEfiles
 stack.info*
 bsg_cadenv/
 *.nbf
+nbf.tar.xz
```

Comment:
Can’t comment on the submodule — is this the SDK before or after the bootrom changes I just pushed today?

Reply:
after, https://github.com/black-parrot-sdk/black-parrot-sdk/tree/49281cc53f924427b40850491b210781d2d9ac46

Reply:
It seems to work fine with the bootrom changes. Do you expect otherwise?

Reply:
Okay. Are there any changes required for FPGA, including vu37/vcu128?

Reply:
Shouldn’t be, as long as the flow is using the NBF file from inside the RTL repo (vu37p does)